packages/pn-personagiuridica-webapp/src/utility/apikeys.utility.tsx

```diff
@@ -61,6 +61,12 @@ export function getApiKeyStatusInfos(
   const tooltip = statusHistory ? <TooltipApiKey history={statusHistory} /> : undefined;
 
   switch (status) {
+    case "Disabled":
```

Comment:
Add "Disabled" state to VirtualKeyStatus enumeration

Reply:
done